.github/workflows/nym_wallet.yml

```diff
@@ -0,0 +1,57 @@
+name: webdriverio tests for nym wallet
+
+on: 
+ push:
+    paths:
+      - '/tauri-wallet/*'
+defaults:
+  run:
+    working-directory: tauri-wallet
+
+jobs:
+  test:
+    name: wallet tests
+    runs-on: ubuntu-latest 
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Tauri dependencies
+        run: >
+          sudo apt-get update &&
+          sudo apt-get install -y
+          libgtk-3-dev
+          libgtksourceview-3.0-dev
+          webkit2gtk-4.0
+          libappindicator3-dev
+          webkit2gtk-driver
+          xvfb
+     
+      - name: Rust stable
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          components: clippy, rustfmt
```

Comment:
I guess there's no point in including `clippy` or `rustfmt` if ultimately they're not used in this flow